### PR TITLE
"Connecting to endpoints" guide - Closes 1583

### LINKED ISSUE #1583: Add `Connecting to endpoints`
### Description

- Add `Connecting to endpoints` to use Lisk SDK v6
- Add all the available ways to call an endpoint (CLI, API client, Curl)

### Page Location
Integrate Blockchain `lisk-docs` -> `development`

### PR DESCRIPTION
**Pull request by TalhaMaliktz**
Closes #1583

### CODE REVIEW

File: docs/modules/ROOT/pages/integrate-blockchain/connecting-endpoints.adoc [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+
+On this page, you'll learn about how the aforementioned endpoints can be invoked within the Lisk ecosystem.
+
+[[the-api-client]]
+== Invoking via API client
+
+// The xref:{url_references_elements_apiclient}[] simplifies sending API requests to a blockchain application. 
+The API client simplifies sending API requests to a blockchain application.
+
+Blockchain applications developed with Lisk SDK communicate to a node via RPCs. RPC-based interfaces i.e. IPC, WS, or HTTP allow applications to *invoke* RPC endpoints.
```
Comment (chris529) [inline docs/modules/ROOT/pages/integrate-blockchain/connecting-endpoints.adoc]:
```suggestion
Blockchain applications developed with the Lisk SDK communicate to a node via RPCs. RPC-based interfaces i.e., IPC, WS, or HTTP allow applications to *invoke* RPC endpoints. 
```

File: docs/modules/ROOT/pages/integrate-blockchain/connecting-endpoints.adoc [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+
+[[the-api-client]]
+== Invoking via API client
+
+// The xref:{url_references_elements_apiclient}[] simplifies sending API requests to a blockchain application. 
+The API client simplifies sending API requests to a blockchain application.
+
+Blockchain applications developed with Lisk SDK communicate to a node via RPCs. RPC-based interfaces i.e. IPC, WS, or HTTP allow applications to *invoke* RPC endpoints. 
+
+An API client can be imported into any JS client application as shown in the following snippets.
```
Comment (chris529) [inline docs/modules/ROOT/pages/integrate-blockchain/connecting-endpoints.adoc:48]:
```suggestion
An API client can be imported into any JS client application as shown in the following snippets below.
```

File: docs/modules/ROOT/pages/integrate-blockchain/connecting-endpoints.adoc [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+        "height": 2291
+    }
+}'
+----
+--
+=====
+
+[TIP]
+====
+Lisk also supports invoking endpoints from within the sidechain client. For more information, check out the xref:{url_sync_store}[syncing and storing] example for the `HelloInfoPlugin`. Specifically, the following:
```
Comment (chris529) [inline docs/modules/ROOT/pages/integrate-blockchain/connecting-endpoints.adoc]:
```suggestion
Lisk also supports invoking endpoints from within the sidechain client. For more information, check out the xref:{url_sync_store}[syncing and storing] example for the `HelloInfoPlugin`. Specifically, as shown in the the following example below:
```

File: docs/modules/ROOT/pages/integrate-blockchain/connecting-endpoints.adoc [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+    }).then(res => {
+        console.log("Result: ", res);
+    });
+});
+----
+--
+HTTP cURL::
++
+--
+Apart from the WS and IPC method, Lisk endpoints also support HTTP requests and response mechanisms. With a {JSON_RPC_Specs}[JSON RPC 2.0] based format, any endpoint can be invoked using a cURL request. For example:
```
Comment (chris529) [inline docs/modules/ROOT/pages/integrate-blockchain/connecting-endpoints.adoc]:
```suggestion
Apart from the WS and IPC method, Lisk endpoints also support HTTP requests and response mechanisms. With a {JSON_RPC_Specs}[JSON RPC 2.0] based format, any endpoint can be invoked using a cURL request. For example, as shown below:
```

**Review by chris529 — COMMENTED** [review]
Looks great! Just a few very minor points 👍

File: docs/modules/ROOT/pages/integrate-blockchain/connecting-endpoints.adoc (hunk 2 above)
Comment (TalhaMaliktz) [reply docs/modules/ROOT/pages/integrate-blockchain/connecting-endpoints.adoc:48]: "Following" and "below" are redundant.

File: docs/modules/ROOT/pages/integrate-blockchain/connecting-endpoints.adoc (hunk 3 above)
Comment (TalhaMaliktz) [reply docs/modules/ROOT/pages/integrate-blockchain/connecting-endpoints.adoc]:
```suggestion
Lisk also supports invoking endpoints from within the sidechain client. For more information, check out the xref:{url_sync_store}[syncing and storing] example for the `HelloInfoPlugin`. Specifically, the line of code shown in the following example:
```

File: docs/modules/ROOT/pages/integrate-blockchain/connecting-endpoints.adoc [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+private async _syncChainEvents(): Promise<void> {
+    // 1. Get the latest block height from the sidechain
+    const res = await this.apiClient.invoke<{ header: { height: number } }>("chain_getLastBlock", {});
+    //[...]
+}
+----
+====
+
+== Invoking via Lisk CLI
+Any endpoint within the Lisk ecosystem can be invoked via Lisk's CLI as well. For more information, check out the xref:{url_cli}[CLI reference page].
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/integrate-blockchain/connecting-endpoints.adoc]: Would add a note for Lisk Core users

```suggestion
Any endpoint within the Lisk ecosystem can be invoked via Lisk's CLI as well.
For more information, check out the xref:{url_cli}[CLI reference page].

NOTE: If you are running *Lisk Core*, you can use nearly the same CLI commands as in the general Lisk CLI.
Just replace `./bin/run` with `lisk-core` in that case.
```

**Review by Tschakki — CHANGES_REQUESTED** [review]
Looks great already! :) Just a small remark.

File: docs/modules/ROOT/pages/integrate-blockchain/connecting-endpoints.adoc (hunk 2 above)
Comment (chris529) [reply docs/modules/ROOT/pages/integrate-blockchain/connecting-endpoints.adoc:48]: Oh yes, is not essential...it's just more personal preference really, and also maintains the description uniformity of the PR, as it is already used in the same fashion in line 154: 
(To invoke an endpoint via Lisk CLI, simply use the `endpoint:invoke` command as shown below:)
Nevertheless, it is your excellent guide Talha, so it is really your call.

**Review by chris529 — APPROVED** [review]
LGTM! 👍

**Review by Tschakki — APPROVED** [review]
LGTM 👍

**Pull request merged**